engine/formatlabels.go

```diff
@@ -0,0 +1,19 @@
+package engine
+
+import (
+	"strings"
+)
+
+// formatLabels takes a comma separated list of agent labels
+// and formats it into map[string]string for agent label matching
+func formatLabels(labels []string) map[string]string {
```

Comment:
is this still being used?

Reply:
no, i was just realizing that myself. i'll pull this out right now